packages/ui/src/components/va-pagination/VaPagination.vue

```diff
@@ -104,16 +104,23 @@ export default defineComponent({
     directionLinks: { type: Boolean, default: true },
     input: { type: Boolean, default: false },
     hideOnSinglePage: { type: Boolean, default: false },
-    flat: { type: Boolean, default: false },
+    plain: { type: Boolean, default: false },
     total: { type: Number, default: null },
     pageSize: { type: Number, default: null },
     boundaryIconLeft: { type: String, default: 'first_page' },
     boundaryIconRight: { type: String, default: 'last_page' },
     directionIconLeft: { type: String, default: 'chevron_left' },
     directionIconRight: { type: String, default: 'chevron_right' },
+    gapped: { type: Boolean, default: false },
+    borderColor: { type: String, default: '' },
+    rounded: { type: Boolean, default: false },
+    currentPageBorderColor: { type: String, default: '' },
   },
 
   setup (props, { emit }) {
+    // TODO(1.6.0): Remove deprecated props
+    useDeprecatedProps(['flat', 'outline'])
```

Comment:
They are already removed. We have decided that we should throw warnings to help understand what to fix after update. We can't remove them during the other update because it's a breaking change.

Reply:
@aluarius, deprecated usually means that it is something that still exists, but not recommended using and will be removed in feature updates.

I think we need just remove these props and write it in migration guide.